Dockerfile

```diff
@@ -30,6 +30,9 @@ WORKDIR ${SONIA_WS}
 
 COPY . ${NODE_PATH}
 
+RUN apt update

```

Comment:
Can you place the lines 33 and 34 before the line 29 so these commands aren't triggered everytime when we change the files of the project